core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -1037,6 +1070,37 @@ private void sendNotificationBroadcast(int type, int code) {
         sendBroadcast(intent);
     }
 
+    private void skipEnding() {
+        Playable playable = mediaPlayer.getPlayable();
+        if (! (playable instanceof FeedMedia)) {
+            return;
+        }
+
+        int duration = getDuration();
+        int remainingTime = duration - getCurrentPosition();
+
+        FeedMedia feedMedia = (FeedMedia) playable;
+        FeedPreferences preferences = feedMedia.getItem().getFeed().getPreferences();
+        int skipEnd = preferences.getFeedSkipEnding();
+        if (skipEnd > 0 &&
+            skipEnd < playable.getDuration() &&
```

Comment:
This PR does not include the more strict diff-checkstyle CI rule. The `&&` characters should be in the next line. Also, indentation should be 8 spaces when breaking the same statement.

You can select the file `config/checkstyle/checkstyle-new-code.xml` in IntelliJ/Android Studio (with checkstyle version 8.18 and "Treat errors as warnings") to get inline suggestions. Alternatively, you can rebase the commit with a more recent `develop` branch to have it executed on CI.